src/components/radar/RadarAnimation.tsx

```diff
@@ -1,88 +1,89 @@
-// @flow
 import * as React from 'react';
 import uniqueId from 'lodash/uniqueId';
 import TetherComponent from 'react-tether';
+import classNames from 'classnames';
 
 import './RadarAnimation.scss';
 
-const BOTTOM_CENTER = 'bottom-center';
-const BOTTOM_LEFT = 'bottom-left';
-const BOTTOM_RIGHT = 'bottom-right';
-const MIDDLE_CENTER = 'middle-center';
-const MIDDLE_LEFT = 'middle-left';
-const MIDDLE_RIGHT = 'middle-right';
-const TOP_CENTER = 'top-center';
-const TOP_LEFT = 'top-left';
-const TOP_RIGHT = 'top-right';
+export enum RadarAnimationPosition {
```

Comment:
A change like this will not break a parent project, since the compiled down es5 version is still string based. However symlinking and using "src" directly will likely no longer work and/or parent projects typescript check will complain even though at runtime it is fine.